### PR TITLE
Updates for multi-channel licenses

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: docs/distributing-an-application/create-licenses.md [hunk 1]
```diff
@@ -20,9 +20,11 @@ If you are looking to create or manage custom license fields you can do so in th
 The name of the customer to whom this license is assigned.
 
 ## Channel (Required)
-When you create a license you'll need to assign it to a release channel (if it is a customer,
+When you create a license you'll need to assign it to at least one release channel (if it is a customer,
```
Comment (wlaoh) [inline docs/distributing-an-application/create-licenses.md]: `if it is a customer, likely channel is Stable` sounds awkward how about `Stable is a likely channel for customers,`  something along those lines.

File: docs/distributing-an-application/create-licenses.md [hunk 2]
```diff
@@ -20,9 +20,11 @@ If you are looking to create or manage custom license fields you can do so in th
...
 likely channel is Stable, but for your internal testing you'll also likely need unstable &
- beta licenses). [More about release channel management](/getting-started/manage-releases/)).
+ beta licenses). [More about release channel management](/getting-started/manage-releases/).
+
+ When license is assigned to multiple channels, the customer will be able to select the channel at install time and later change the release channel in management console.  For airgapped installs, the channel can be selected at download time only.
```
Comment (wlaoh) [inline docs/distributing-an-application/create-licenses.md]: `When a license`, `release channel in the management console`

File: docs/kb/supporting-your-customers/multichannel-licenses.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+lastmod = "2017-06-12T00:00:00Z"
+title = "Multi-channel Licenses"
+weight = "999999"
+categories = [ "Knowledgebase", "Supporting Your Customers" ]
++++
+
+## Installing Multi-Channel License
+When multiple release channels are assigned to a license, the customer will be able to select the channel during installation time.
+
+![Invalid License](/static/license-upload-channels.png)
```
Comment (wlaoh) [inline docs/kb/supporting-your-customers/multichannel-licenses.md]: Why is this named `Invalid License`?

File: docs/kb/supporting-your-customers/multichannel-licenses.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+* Channel drop-down will show all channels assigned to the license.
+* Channel description, located under the drop-down, can be edited on the Channels screen in the Replicated Vendor Portal.
+* Current version tag and release notes can be modified on the Channel History screen in the Replicated Vendor Portal.
+
+### Airgapped Installations
+For airgapped installations, license channel has to be selected at the time when the airgap bundle is downloaded.
```
Comment (wlaoh) [inline docs/kb/supporting-your-customers/multichannel-licenses.md]: `the license channel`

File: docs/kb/supporting-your-customers/multichannel-licenses.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+## Changing Channel
+Channel can be changed on the License page in Replicated management console.  Once channel is  changed, Replicated will sync license and check for release updates.  If newer releases are available, the customer will be able to upgrade the application.
```
Comment (wlaoh) [inline docs/kb/supporting-your-customers/multichannel-licenses.md]: Extra space between `is` and `changed`

**Review by wlaoh — CHANGES_REQUESTED** [review]

File: docs/kb/supporting-your-customers/multichannel-licenses.md (hunk 5 above)
Comment (divolgin) [reply docs/kb/supporting-your-customers/multichannel-licenses.md]: Who cares! It's HTML! :)

**Pull request merged**